src/02-common-principles.md

```diff
@@ -650,6 +650,16 @@ it refers to. For example `sub-01` refers to the `sub` entity (a
 subject) with the label `01`. The `sub-` prefix is not part of the subject
 label, but must be included in file names (similarly to other key names).
 
+## Specification of paths
+
+Several metadata fields in BIDS require the specification of paths, that is
+a string of characters used to uniquely identify a location in a directory
+structure. For example the `IntendedFor` or `AssociatedEmptyroom` metadata fields.
+
+Throughout BIDS all such paths MUST be specified using the slash character (`/`),
+regardless of the operating system that a particular dataset is curated on
+or used on.
```

Comment:
I guess if such section is added, it should specify that paths must not be absolute local paths, thus must not **start** with `/`. 
Didn't analyze yet, but also something might need to be said then what they are relative to, or that what they are relative to would depend on where used.

Reply:
I feel like all of this would be part of #820 so maybe this PR should be adjusted and merged AFTER #820 is in?

Reply:
I added some language for that, ... in the meantime, #820 was replaced with #918 - so we should either:

1. merge this before #918 and then potentially adjust the added text here in #918, **OR**
2. wait until #918 is merged, then adjust this one and merge it.